plugins/models/resnet18.py

```diff
@@ -0,0 +1,67 @@
+import keras
+import keras_resnet
+import keras_resnet.models
+import tensorflow as tf
+
+from deepprofiler.learning.model import DeepProfilerModel
+
+
+def define_model(config, dset):
+
+    def make_regularizer(transforms, reg_lambda):
```

Comment:
The regularizer can be dropped. Can you remove this function? We will not support this type of regularization going forward.